src/SoCreate.Extensions.Logging/LoggerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.Logging;
+
+namespace SoCreate.Extensions.Logging
+{
+    public static class LoggerExtensions
+    {
+        public static void LogSecurity(this ILogger logger, int errorNumber, string message)
+        {
+            logger.LogWarning("{LogType} - {ErrorNumber}: {Message}", "Security", errorNumber, message);
+        }
+
+        public static void LogAudit(this ILogger logger, int errorNumber, string message)
+        {
+            logger.LogInformation("{LogType} - {ErrorNumber}: {Message}", "Audit", errorNumber, message);
+        }
+    }
+}
```

Comment:
I not sure we want to have this file. It was really just an example and we have not discussed if this is something we want.

Reply:
Remove this.